server/publicapi/tests/datalayer_test.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; -*-
+#
+# This file is part of Superdesk.
+#
+# Copyright 2013, 2014, 2015 Sourcefabric z.u. and contributors.
+#
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+
+from publicapi.tests import ApiTestCase
+
+
+class DatalayerTestCase(ApiTestCase):
+    """Base class for `publicapi.datalayer.ApiDataLayer` class tests."""
+
+    def setUp(self):
+        try:
+            from publicapi.datalayer import ApiDataLayer
+        except ImportError:
+            # a missing class should result in a test failure, not in an error
+            self.fail("Could not import class under test (ApiDataLayer).")
+        else:
+            # XXX: do we need to pass the app object here? or is None fine?
```

Comment:
`eve.io.base.DataLayer` (the grandparent of the class under test) expects an (optional) Flask application instance as a [parameter to `__init__`](https://github.com/nicolaiarocci/eve/blob/develop/eve/io/base.py#L94). It's a pattern to extend Flask.

I am not yet sure whether this is needed for the test purposes or not, or is passing `None` (or not passing anything) just fine, hence the comment.